### PR TITLE
JSTL - InstructorFeedbackResponseCommentsAdd

### PR DESCRIPTION
**Pull request by junhaoyap**
Still in process

### CODE REVIEW

**Comment by wkurniawan07** [comment]
Nice, wait until my #3863. It's close to finished now.

**Comment by junhaoyap** [comment]
Ah oops ok I'll go do other stuff first then, thought it was done already >.<

**Comment by wkurniawan07** [comment]
Can continue. My FRC tag should be perfectly reusable.

**Comment by junhaoyap** [comment]
Ok! I'll continue, most likely more deletions than additions in this PR when I reuse your tag.

**Comment by junhaoyap** [comment]
I'll run the tests during dinner before pinging you for a review @yj-soh

File: src/main/webapp/WEB-INF/tags/shared/feedbackResponseComment.tag [hunk 1]
```diff
@@ -3,6 +3,8 @@
 <%@ tag import="teammates.common.util.Const" %>
 <%@ tag import="teammates.common.datatransfer.FeedbackParticipantType" %>
 <%@ attribute name="frc" type="teammates.ui.template.FeedbackResponseComment" required="true" %>
+<%@ attribute name="commentId" %>
```
Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/shared/feedbackResponseComment.tag]: Is this still needed?

File: src/main/webapp/WEB-INF/tags/shared/feedbackResponseComment.tag [hunk 2]
```diff
@@ -132,127 +134,117 @@
                                        <c:if test="${frc.showGiverNameToResponseGiver}">checked="checked"</c:if>>
                             </td>
                         </tr>
-                        <c:if test="${frc.responseVisibleToRecipient}">
```
Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/shared/feedbackResponseComment.tag]: I didn't put these tests in the tag for no reason. Check instructorFeedbackResultsByGRQ or ByRGQ ("those things") to see what I mean.

Comment (junhaoyap) [inline src/main/webapp/WEB-INF/tags/shared/feedbackResponseComment.tag]: Ok I'll take a look at them tomorrow, I wasn't sure what they were used for and I didn't want to instantiate variables that wouldn't be used elsewhere, I'll take a look first thing tomorrow and make the appropriate changes, thanks for pointing me to the right direction!

File: src/main/webapp/WEB-INF/tags/shared/feedbackResponseComment.tag (hunk 1 above)
Comment (junhaoyap) [inline src/main/webapp/WEB-INF/tags/shared/feedbackResponseComment.tag]: Nope! Thanks for the catch!

**Comment by junhaoyap** [comment]
I'll update tomorrow morning, I just managed to get home after the train disruption...

**Comment by junhaoyap** [comment]
I'll update the branch and point it for merge into master instead so I'll be closing this!

**Pull request closed without merging**